BlockLimiter/Patch/BuildBlockPatch.cs

```diff
@@ -121,28 +121,32 @@ private static bool BuildBlocksRequest(MyCubeGrid __instance, HashSet<MyCubeGrid
 
         private static bool Build(MyProjectorBase __instance, Vector3I cubeBlockPosition, long owner, long builder, bool requestInstant = true, long builtBy = 0)
         {
-            if (__instance == null) return false;
+            var projector = __instance;
+            if (projector == null) return false;
 
             if (!BlockLimiterConfig.Instance.EnableLimits) return true;
 
-            var grid = __instance?.CubeGrid;
-
-            if (grid == null) return false;
+            if (owner + builder == 0) return false;
```

Comment:
Do you mean it should return true to support build & repair mods? I kept the return false from your original code.

Reply:
if grid is null, the entire code shouldn't run and the game is acting up with a phantom build, but owner and builder can be 0 because of reasons.

Reply:
Build & Repair has just reported to be working with MGP Torch plugin 0.1.19 and modified Block Limiter (according to this patch).